Translations/hr.po

```diff
@@ -1203,12 +1203,12 @@ msgstr ""
 
 #: Source/stores.cpp:1129
 #, c-format
-msgid "Talk to %s"
+msgid "Talk to {}"
 msgstr ""
 
 #: Source/stores.cpp:1133
 #, c-format
```

Comment:
Is `c-format` still the correct thing here?

Reply:
Not sure, also not sure how xgettext determines this :thinking:

Reply:
> Is `c-format` still the correct thing here?

I don't think so. poedit removed that when I save a test file with translations using {}.